kore/app/share/GlobalMain.hs

```diff
@@ -200,17 +205,34 @@ data KoreMergeOptions =
     KoreMergeOptions
         { rulesFileName     :: !FilePath
         -- ^ Name for file containing a sequence of rules to merge.
+        , maybeBatchSize    :: Maybe Int
         }
 
 parseKoreMergeOptions :: Parser KoreMergeOptions
 parseKoreMergeOptions =
     KoreMergeOptions
     <$> strOption
-            (  metavar "MERGE_RULES_FILE"
-            <> long "merge-rules"
-            <> help
-                "List of rules to merge."
-            )
+        (  metavar "MERGE_RULES_FILE"
+        <> long "merge-rules"
+        <> help
+            "List of rules to merge."
+        )
+    <*> option
+        maybeIntReader
+        (  metavar "MERGE_BATCH_SIZE"
+        <> long "merge-batch-size"
+        <> help
+            "The size of a merge batch."
```

Comment:
How do we say "all the rules"? Is it `--merge-batch-size 0`?

Reply:
Wow, that was fast! If you don't mention --merge-batch-size it should merge all rules at once, but I didn't test it yet

Reply:
Hm, my message below does not seem to be attached to this one. Oh well...

This merged the rule list you last sent me in ~6 seconds. I'm running it like this:
```
kore-exec definition.kore --merge-rules rule-list --module WASM-WITH-K-TERM --merge-batch-size=2
```
It produces two rather large rules, but they seem reasonable at first sight (I mostly looked at their "shape", not at their contents).

Reply:
`--merge-batch-size 0` merges all rules at once.